woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -317,21 +317,16 @@ def test_accessor_with_numeric_time_index(time_index_df):
     with pytest.raises(TypeError, match=error_msg):
         time_index_df.ww.init(time_index='letters', logical_types={'strs': 'Integer'})
 
-    # --> add back when schema updates are implemented
-    # # Change time index to normal datetime time index
-    # schema = schema.set_time_index('times')
-    # date_col = schema['ints']
-    # assert schema.time_index == 'times'
-    # assert date_col.logical_type == Double
-    # assert date_col.semantic_tags == {'numeric'}
-
-    # Set numeric time index after init
-    # schema = Schema(time_index_df, logical_types={'ints': 'Double'})
-    # schema = schema.set_time_index('ints')
-    # date_col = schema['ints']
-    # assert schema.time_index == 'ints'
-    # assert date_col.logical_type == Double
-    # assert date_col.semantic_tags == {'time_index', 'numeric'}
+    # Change time index to normal datetime time index
```

Comment:
Comment wrong? Should this be `Set numeric time index after init` instead?

Reply:
yep! Changing